Laud_Misc_610/CGH/_1.trig

```diff
@@ -76,14 +80,14 @@
 <#Follaig>
     a foaf:Person;
     irishRel:nomName "Follaig";
-    rel:childOf <#Hetherél>;
+    rel:childOf <#Hirélfáith>;
```

Comment:
Why have you made Follaig son of Iriel (Hirél), not son of Ethriel (Hetherél)? The text is not very clearly worded at this point, but Ethriel is the last person mentioned:

> Ní fárcaib mac díb claind act óen mac .i. Hirél fáith .i. Hetherél mac Hirél. Dá mac lais .i. Follaig & Foliaiach ...

Having Follaig as the son of Ethriel also fits in with other sources; e.g. "... Fallaig m. Ethréoil m. h-Íréoil Fátha m. h-Érimóin", from _[Genealach Osrithe](https://celt.ucc.ie//published/G105003/text002.html)_.